x-pack/plugins/security_solution/public/common/components/callouts/callout.test.tsx

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { mount } from 'enzyme';
+import React from 'react';
+import { CallOut, CallOutMessage } from '.';
```

Comment:
Let's maybe import from `./callout_types` to avoid circular deps?

Reply:
Yeah I can do that.